examples/logger_config.yml

```diff
@@ -34,6 +34,9 @@ handlers:
         stream: ./example_error.log
 
 loggers:
+    loggerA.child:
+        handlers: [console]
```

Comment:
Same as above. It'll be more clear to novice users if we use different handlers in the examples for child and parent loggers.